tutorials/how_to/elmo.md

```diff
@@ -0,0 +1,161 @@
+
+Using pre-trained ELMo representations
+--------------------------------------
+
+Pre-trained contextual representations from large scale bidirectional
+language models provide large improvements for nearly all supervised
+NLP tasks.
+
+This document describes how to add ELMo representations to your model using `allennlp`.
+We also have a tensorflow implementation [here](https://github.com/allenai/bilm-tf).
+
+Reference: ["Deep contextualized word representations"](https://openreview.net/forum?id=S1p31z-Ab)
```

Comment:
`For more detail about ELMo, please see the publication ["Deep contextualized word representations"](https://openreview.net/forum?id=S1p31z-Ab).`